src/js/edu-benefits/components/military-history/ContributionsFields.jsx

```diff
@@ -65,23 +61,19 @@ export default class ContributionsFields extends React.Component {
               checked={this.props.data.activeDutyRepaying}
               onValueChange={(update) => {this.props.onStateChange('activeDutyRepaying', update);}}/>
           <div>
-            <DateInput required
-                errorMessage="Please provide a response"
-                validation={validateIfDirtyDateObj(this.props.data.activeDutyRepayingPeriod.from, isValidDateField)}
+            <ErrorableCurrentOrPastDate required
                 label="Start date"
                 name="from"
-                day={this.props.data.activeDutyRepayingPeriod.from.day}
-                month={this.props.data.activeDutyRepayingPeriod.from.month}
-                year={this.props.data.activeDutyRepayingPeriod.from.year}
+                date={this.props.data.activeDutyRepayingPeriod.from}
                 onValueChange={(update) => {this.props.onStateChange('activeDutyRepayingPeriod.from', update);}}/>
-            <DateInput required
-                errorMessage={isToDateValid ? 'End Date must be after Start Date' : 'Please provide a response'}
-                validation={validateIfDirtyDateObj(this.props.data.activeDutyRepayingPeriod.to, (date) => isValidDateRange(this.props.data.activeDutyRepayingPeriod.from, date))}
+            <ErrorableCurrentOrPastDate required
+                validation={{
+                  valid: isValidDateRange(this.props.data.activeDutyRepayingPeriod.from, this.props.data.activeDutyRepayingPeriod.to),
+                  message: 'End date must be after Start date'
```

Comment:
I think I did that because it's a field name, but it seems generic enough that it's not necessary.